api/bin/spark.js

```diff
@@ -23,6 +24,9 @@ const {
 // The same token is configured in Fly.io secrets for the deal-observer service too.
 assert(DEAL_INGESTER_TOKEN, 'DEAL_INGESTER_TOKEN is required')
 
+// TODO:
+assert(CHECKER_TOKEN, 'CHECKER_TOKEN is required')
```

Comment:
What is this TODO for?

Reply:
Good catch, I've left TODO to add adittional context why is the `CHECKER_TOKEN` required.